### PR TITLE
[CMR] M3-4358: Table styles for Profile > SSH Keys

### PR DESCRIPTION
**Pull request by tiffwong**
Something easy to work on :)

### CODE REVIEW

**Comment by tiffwong** [comment]
> Is there a plan for responsive stuff for these? there's no detail view for keys so information would be lost, but otoh scrolling to the delete button is annoying

@Jskobos What do you think of hiding the "Created" column for mobile view?

File: packages/manager/src/features/Profile/SSHKeys/SSHKeys_CMR.tsx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+  }
+
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <div
+        id="tabpanel-sshKeys"
+        className={classes.root}
+        role="tabpanel"
```
Comment (WilkinsKa1) [inline packages/manager/src/features/Profile/SSHKeys/SSHKeys_CMR.tsx:125]: we can remove the role, aria, and id attributes- reach determines these for us!

File: packages/manager/src/features/Profile/SSHKeys/SSHKeys_CMR.tsx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+      margin: 0,
+      width: '100%'
+    },
+    headline: {
+      marginTop: 8,
+      marginBottom: 8,
+      marginLeft: 15,
+      lineHeight: '1.5rem',
+      [theme.breakpoints.down('xs')]: {
+        marginBottom: 0,
```
Comment (WilkinsKa1) [inline packages/manager/src/features/Profile/SSHKeys/SSHKeys_CMR.tsx:53]: These styles cause the heading to be misaligned with the button:
Screen Shot 2020-08-10 at 10 53 12 AM

Comment (WilkinsKa1) [reply packages/manager/src/features/Profile/SSHKeys/SSHKeys_CMR.tsx:53]: I think if you remove everything should be aligned

**Pull request merged**